rmgpy/yml.py

```diff
@@ -0,0 +1,168 @@
+import yaml
+import os
+from rmgpy.chemkin import loadChemkinFile
+from rmgpy.species import Species
+from rmgpy.reaction import Reaction
+from rmgpy.thermo.nasa import NASAPolynomial, NASA
+from rmgpy.thermo.wilhoit import *
+from rmgpy.kinetics.arrhenius import *
+from rmgpy.kinetics.falloff import *
+from rmgpy.kinetics.chebyshev import *
+from rmgpy.data.solvation import *
+from rmgpy.util import makeOutputSubdirectory
+
+def convertchemkin2yml(chemkinpath,spcdictpath=None,output="chem.rms"):
+    if spcdictpath:
+        spcs,rxns = loadChemkinFile(chemkinpath,dictionaryPath=spcdictpath)
+    else:
+        spcs,rxns = loadChemkinFile(chemkinpath)
+    writeyml(spcs,rxns,path=output)
+
+def writeyml(spcs,rxns,solvent=None,solventData=None,path="chem.yml"):
+    D = getmechdict(spcs,rxns,solvent=solvent,solventData=solventData)
+    yaml.dump(D,stream=file(path,'w'))
+
+def getmechdict(spcs,rxns,solvent='solvent',solventData=None):
+    D = dict()
+    D["Units"] = dict()
+    D["Reactions"] = []
+    D["Phases"] = [dict()]
+    D["Phases"][0]["name"] = "phase"
+    D["Phases"][0]["Species"] = [obj2dict(x,spcs) for x in spcs]
+    D["Reactions"] = [obj2dict(x,spcs) for x in rxns]
+    if solventData:
+        D["Solvents"]= [obj2dict(solventData,spcs,label=solvent)]
+    return D
+
+def getradicals(spc):
+    if spc.molecule[0].toSMILES() == "[O][O]":
```

Comment:
What is the purpose of this function? Can you not use Molecule.getRadicalCount?

Also, why does oxygen need to be special cased? Could you add a comment explaining?

Reply:
I think this is faster since the multiplicities are known.  

This is typically used for analyzing radical production in RMS and it's very useful to treat O2 as a stable molecule when you're looking at bulk radical production/loss